accepted/2020/workloads/workload-manifest.md

```diff
@@ -0,0 +1,430 @@
+        "Xamarin.Android.Runtime": {
+            "kind" : "framework",
+            "version": "8.4.7.4"
+        },
+        // targets and tools for taking IL assemblies and producing a set of
+        // binaries that can run on android. this comes from dotnet/runtime,
+        // and its tasks are invoked by the Xamarin.Android.Sdk targets that
+        // handle packing and bindings-related tasks
+        "Mono.Android.Sdk": {
+            "kind" : "sdk",
+            "version": "7.0.1"
+        },
+        // runtime binaries for x86 devices
+        "Mono.Android.Runtime.x86": {
+            "kind" : "framework",
+            "version": "7.0.1"
+        },
+        // runtime binaries for x86 devices
+        "Mono.Android.Runtime.Armv7a": {
+            "kind" : "framework",
+            "version": "7.0.1"
+        },
+        // build tools for Android that include native binaries that
+        // are specific to the host platform.
+        "Xamarin.Android.BuildTools": {
+            "version" : "8.4.7",
+            "kind": "sdk",
+            "alias-to": {
+                "osx-x64": "Xamarin.Android.BuildTools.MacHost",
+                "win-x64": "Xamarin.Android.BuildTools.Win64Host"
+            }
+        },
+        // this is also has host specific binaries. although it is an
+        // "sdk" pack the MSBuild logic might be trivial and simply
+        // set properties with the compiler location to be used by
+        // the targets in Mono.Android.Sdk
+        "Mono.Android.LLVM.Aot.armv7a": {
+            "version" : "8.4.7",
+            "kind": "sdk",
+            "alias-to": {
+                "osx-x64": "Mono.Android.LLVM.Aot.armv7a.MacHost",
+                "win-x64": "Mono.Android.LLVM.Aot.armv7a.Win64Host"
+            }
+        }
+    }
+}
+```
+
+# Side-by-Side Platform Updates
+
+In .NET 5, the [TFM is used to specify the .NET version](https://github.com/dotnet/designs/blob/master/accepted/2020/net5/net5.md) (e.g. .NET 5, .NET 6), and may also be used to specify a target platform (e.g. iOS, Android, WinUI) to access platform-specific API. If the TFM specifies a platform, it also implicitly or explicitly specifies a *Platform API Version*. Developers are generally expected to use the latest platform API version, specify a minimum OS version, and [use runtime checks to guard use of APIs from newer OS versions](minimum-os-version/minimum-os-version.md). Templates burn in the most recent OS API version so that new projects start out in the recommended configuration, warning-free and with access to the latest APIs.
+
+It is strongly recommended that updates *within* an SDK feature band only contain low-risk servicing updates, not new features. This means that developers can feel safe installing servicing updates. They know that it's not going to allow them to accidentally make the project depend on APIs that will break it for other members on their team who are not as up to date.
```

Comment:
We should strengthen this. We already have a very strong policy on minimal churn low-risk servicing updates, and it's not up for debate. However, there are some new, very specific, scenarios where we may need to relax this policy. Obviously, those need to be considered with an understanding of risk, too, but I don't think that's a key issue.